src/stage1/parse_f128.c

```diff
@@ -10,6 +10,15 @@
 #include <string.h>
 #include <math.h>
 
+// Every OSes seem to define endianness macros in different files.
```

Comment:
*cough* Windows

Reply:
My bad. I can't seem to find an equivalent for endian.h or __BYTE_ORDER and friends there, though.
How should I check for byte ordering there? Or is it safe to assume that Windows will always use LE ordering?

Reply:
> My bad.

Don't worry, finding the correct headers across this many platforms is hard.

> Or is it safe to assume that Windows will always use LE ordering?

That's a safe guess.

Reply:
Okay, added a definition for Windows.

Reply:
I'm afraid BSDs use `_TARGET_ENDIAN`... See [here](https://github.com/freebsd/freebsd/blob/a724ee826279ecbcf636cbb0125b23097ec6f068/sys/x86/include/endian.h#L55) :\

You're better off creating a new macro, doing all this renaming is ugly and error prone.

```C
#if defined(__APPLE__)
  #include <machine/endian.h>
  #define ENDIAN_LE BYTE_ORDER == LITTLE_ENDIAN
#elif defined(__DragonFly__) || defined(__FreeBSD__) || defined(__NetBSD__) || defined(__OpenBSD__)
  #include <sys/endian.h>
  #define ENDIAN_LE _BYTE_ORDER == _LITTLE_ENDIAN
#elif defined(_WIN32) || defined(_WIN64)
  #define ENDIAN_LE 1
#else 
  #include <endian.h>
  #define ENDIAN_LE __BYTE_ORDER == __LITTLE_ENDIAN
#endif
```